python/tests/integration/arcticdb/version_store/test_basic_version_store.py

```diff
@@ -1725,7 +1726,11 @@ def test_dynamic_schema_column_hash_update(lmdb_version_store_column_buckets):
     lib.update("symbol", df2)
     vit = lib.read("symbol")
     df.update(df2)
-    assert_frame_equal(vit.data.astype("float"), df)
+    if not IS_PANDAS_TWO:
+        # Pandas 1.0 does not cast to float natively, but Pandas 2.0 does.
+        vit.data = vit.data.astype("float")
```

Comment:
Is there any harm in doing this in Pandas 2 as well?

I don't want to give any impression that there's Pandas version-dependent behaviour in this test

Reply:
In Pandas < 2.0, updating a `DataFrame` uniquely storing integers with another `DataFrame` that is uniquely storing integers changes the dtypes to float64 (yes).

This is fixed in Pandas >= 2.0.

The comment is not accurate, and I will update it.

Reply:
Updated in 97928e1.